system_metrics_collector/src/system_metrics_collector/linux_memory_measurement_node.cpp

```diff
@@ -124,4 +125,30 @@ double LinuxMemoryMeasurementNode::periodicMeasurement()
   return processMemInfoLines(read_string);
 }
 
+void LinuxMemoryMeasurementNode::publishStatistics()
+{
+  if (publisher_ == nullptr) {
+    return;
+  }
```

Comment:
ditto, throw in unallowed

Reply:
Removed this check protected/private member function.